tests/ast/ast_test_utils.py

```diff
@@ -68,6 +72,76 @@ def generate_error_trace_info(python_text, exception=None):
     return error_msg
 
 
+def is_valid_ast_obj(ast):
+    return isinstance(ast, Iterable) or (
```

Comment:
what does "valid" mean here? Can we add a docstring to state what we interpret as "valid"?

Reply:
No longer used so removed.